advanced/project-importing.md

```diff
@@ -0,0 +1,86 @@
+# Project Importing
+
+Mimic app can import projects from external file, this feature could be very useful and it is very easy to use.
```

Comment:
I'd add an example here to explain why this feature is usefule

Reply:
I added an example on the next paragraph, maybe I can explain that with the file in project folder you can share it through GIT with other team members